codec_specs.md

```diff
@@ -764,3 +784,70 @@ Codec Name: VobBtn Buttons
 
+## Block Addition Mappings
+
+Registered `BlockAddIDType` are:
+
+### Use BlockAddIDValue
+
+Block type identifier: 0
+
+Block type name: Use BlockAddIDValue
+
+Description: This value indicates that the actual type is stored in `BlockAddIDValue` instead. This value is expected to be used when it is important to have a strong compatibility with players or derived formats not supporting `BlockAdditionMapping` but using `BlockAdditions` with an unknown `BlockAddIDValue`, and SHOULD NOT be used if it is possible to use another value.
+
+### Opaque data
+
+Block type identifier: 1
+
+Block type name: Opaque data
+
+Description: the `BlockAdditional` data is interpreted as opaque additional data passed to the codec with the Block data. `BlockAddIDValue` MUST be 1.
+
+### ITU T.35 metadata
+
+Block type identifier: 4
+
+Block type name: ITU T.35 metadata
+
+Description: the `BlockAdditional` data is interpreted as ITU T.35 metadata as defined by ITU-T T.35 terminal codes. `BlockAddIDValue` MUST be 4.
+
+### avcE
+
+Block type identifier: 0x61766345
+
+Block type name: Dolby Vision enhancement-layer AVC configuration
+
+Type Description: the `BlockAddIDExtraData` data is interpreted as `AVCDecoderConfigurationRecord` structure for the enhancement layer as defined in [@!ISO.14496-15.2014], as described in [Dolby Vision Streams File Format](ttps://www.dolby.com/us/en/technologies/dolby-vision/dolby-vision-bitstreams-within-the-iso-base-media-file-format-v2.1.2.pdf). This extension MUST NOT be used if `Codec ID` is not `V_MPEG4/ISO/AVC`.
+
+### dvcC
+
+Block type identifier: 0x64766343
+
+Block type name: Dolby Vision configuration
+
+Description: the `BlockAddIDExtraData` data is interpreted as `DOVIDecoderConfigurationRecord` structure as defined in [Dolby Vision Streams File Format](https://www.dolby.com/us/en/technologies/dolby-vision/dolby-vision-bitstreams-within-the-iso-base-media-file-format-v2.1.2.pdf), for Dolby Vision profiles less than and equal to 7.
+
+### dvvC
+
+Block type identifier: 0x664767643
+
+Block type name: Dolby Vision configuration
+
+Description: the `BlockAddIDExtraData` data is interpreted as `DOVIDecoderConfigurationRecord` structure as defined in [Dolby Vision Streams File Format](https://www.dolby.com/us/en/technologies/dolby-vision/dolby-vision-bitstreams-within-the-iso-base-media-file-format-v2.1.2.pdf), for Dolby Vision profiles greater than 7.
+
+### hvcE
+
+Block type identifier: 0x68766345
+
+Block type name: Dolby Vision enhancement-layer HEVC configuration
+
+Type Description: the `BlockAddIDExtraData` data is interpreted as `HEVCDecoderConfigurationRecord` structure for the enhancement layer as defined in [@!ISO.14496-15.2014], as described in [Dolby Vision Streams File Format](ttps://www.dolby.com/us/en/technologies/dolby-vision/dolby-vision-bitstreams-within-the-iso-base-media-file-format-v2.1.2.pdf). This extension MUST NOT be used if `Codec ID` is not `V_MPEGH/ISO/HEVC`.
```

Comment:
Same remark on the reference vs link.

Reply:
Does that mean we have the same `HEVCDecoderConfigurationRecord` twice in the `CodecPrivate` and in the `hvcE` ?

Reply:
I forgot this one, `hvcE` contains `HEVCDecoderConfigurationRecord` of the **enhancement layer**, it is not a duplicate of the one in `CodecPrivate` (at least in its definition). It is described in the Dolby doc.
IIRC stuff like color info may is usually different i.e. nothing incompatible with base layer, but in practice whatever is the content of `hvcE`, it is mandatory by Dolby specs so should be in the track header.

Reply:
> data is interpreted as `HEVCDecoderConfigurationRecord` structure for the enhancement layer as defined in [@!ISO.14496-15.2014], as described in [@!DolbyVisionWithinIso.2020-02]

IMO the wording is not very clear. It doesn't show the difference with

> contains a `HEVCDecoderConfigurationRecord` structure as defined in [@!ISO.14496-15.2014].

I propose 

> data contains the enhancement layer of `HEVCDecoderConfigurationRecord` from [@!ISO.14496-15.2014]  as described in [@!DolbyVisionWithinIso.2020-02]

Reply:
@robUx4 your proposal is for me less clear, as it is not an "enhancement layer of `HEVCDecoderConfigurationRecord`", it is a `HEVCDecoderConfigurationRecord` structure, but not for the "classic" stream (it is for the Dobly Vision additions).

In order to be less controversial and for more difference with the `CodecPrivate` part, I modified the PR with the exact wording from the [Dolby spec section 2.3](https://www.dolby.com/us/en/technologies/dolby-vision/dolby-vision-bitstreams-within-the-iso-base-media-file-format-v2.1.2.pdf#_OPENTOPIC_TOC_PROCESSING_d100e6125) for the description of the data : "data is interpreted as the Dolby Vision enhancement-layer HEVC configuration as described in [@!DolbyVisionWithinIso.2020-02]", this way there is no more direct reference to `HEVCDecoderConfigurationRecord` (just an indirect one through the Dolby Vision spec) so no misinterpretation possible, and it may be more future proof (we describe nothing in the content here, the Dolby spec is the only source for the description of the content).

I understand this is the only change request, so please review again.

Reply:
LGTM. I was going to propose dropping the HEVCDecoderConfigurationRecord from the text to make it clearer but I wasn't sure.